### DISCUSSION TITLE
Revert the change to converted bonus money from XP for quests.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#7056] [Suggestions]
Currently there is a 0.5 multiplier on the money rewarded from quests as a result of converting XP to copper (6c = 1XP becomes 3c = 1XP). In practice, this looks like this:

Take the quest `Banish More Demons` for example. On both Blizz and CC, the quest yields 4g, 40s as a quest reward as well as 12,650 XP. Where the two differ is the XP conversion to copper. On Blizz, the conversion would yield 7g, 59s, so 11g, 99s in total at max level. However, on CC the same conversion yields 3g, 79s, 50c, for a total of 8g, 19s, 50c.

This means that the CC quest yields in total a bit under 70% (if my math is right) of what it should at max level. This made sense back in vanilla content as gold starving people sort of fit the feel of vanilla, but as we get closer and closer to Wrath content, this change only hurts players who are doing the content (daily quests or loremasters) for no tangible gain. I suggest that the multiplier be brought back to its default value of 1. If this suggestion is simply too ludicrous to be considered, at least add this change to the [rates and changes](https://www.chromiecraft.com/en/rates-n-changes/) page so it's not blatantly incorrect anymore.

**Comment by UnePomme** [comment]
I personally avoid quests at max level, exactly because of the lower rate.
This would be a great change.

**Comment by heyitsbench** [comment]
https://discord.com/channels/788063059926712341/794247581763895316/1284270991740108830 Seems this has been accepted.
